autocertkit/testbase.py

```diff
@@ -246,6 +246,14 @@ def generate_static_net_conf(self):
                         res[key_name] = sm
                         log.debug("Added static conf for '%s'" % key_name)
 
+        mgmt = get_pool_management_device(self.session)
+        if 'static_management' in netconf and mgmt not in netconf:
+            log.debug("Create static config for mgmt device %s" % mgmt)
+            key_name = "%s_0" % (mgmt)
+            assert(key_name not in res.keys())
```

Comment:
I think in this case it would be nicer to handler the error with a friendly error message. I can for see people adding both the 'static_management' entry and the normal 'eth0_0' entry by mistake - it would be nice to make it clear that they can't do that.

What do you think?